WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -479,6 +485,8 @@ class PostCoordinator: NSObject {
 
     private func startSync(for post: AbstractPost) {
         guard let revision = post.getLatestRevisionNeedingSync() else {
+            let worker = getWorker(for: post)
```

Comment:
If you change the publicly visible upload state, it should also send `.postDidUpdateNotification`.

Reply:
Done https://github.com/wordpress-mobile/WordPress-iOS/pull/22992/commits/522b57160de80a3823edbc177ed1b6871e1695d4